Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM eclipse-temurin:17-jdk-jammy
+
+ENV NODE_VERSION="18.7"
+ENV SYSON_VERSION="2024.1.0"
```

Comment:
I tried to build the container and the value of this env var is wrong. I'd say it must be `2024.3.0`

Reply:
Indeed, this PR has been provided before the release of 2024.3.0.
The author never take into account my comments since then.

Reply:
Hi @AxelRICHARD , when I manually fix this in the running container i still get a startup error (see below).
Can you help and tell me what I or the Dockerfile misses here?

Where do I see your comments?

![image](https://github.com/eclipse-syson/syson/assets/45113557/898edad4-5d35-49d5-abac-4d41ff609299)